### PR TITLE
main color and font variables in sass, skeleton positions

### PR DESCRIPTION
**Pull request by jualiasha**

### CODE REVIEW

File: tools/assets/sass/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+$text-color: #ecebff; //almost white
```
Comment (Romkin7) [inline tools/assets/sass/_variables.scss]: Look how to setup variables file from my devdesign.fi site


/* Font */
$sourceSansPro: "Source Sans Pro", sans-serif;

/* Colors */
/* Primary colors */
$violet: #221e56;
$midnightblue: #0f0868;
$ligtermidnightBlue: #2b2a73;
$hotpink: #e55cb8;
$lightpink: #f27495;
$lavender: #e0e5f0;
$goldenrod: #e4ba0a;
$orangishRod: #f26b63;

/* Gray colors */
$gray100: #f8f8f8;
$gray200: #eeeeee;
$gray300: #e8e8e8;
$gray900: #434343;

/* Black and White colors */
$black: #000000;
$white: #ffffff;
$transparent: transparent;

/* Gradients */
$pinkLinearGradientToRight: linear-gradient(to right, $lightpink, $hotpink);
$orangeLinearGradientToRight: linear-gradient(
  to right,
  $goldenrod,
  $orangishRod
);
$pinkLinearGradientToRight80: linear-gradient(
  to right,
  rgba($lightpink, 0.8),
  rgba($hotpink, 0.8)
);
$radialGradientToCenter: linear-gradient($midnightblue, $ligtermidnightBlue);

/** Borders */
$grayBorder: 4px solid rgba(255, 255, 255, 0.5);
$darkGrayBorder: 2px solid rgba($gray300, 0.8);

/** Shadows */
$softShadow: 0 0 10px 5px rgba(0, 0, 0, 0.2);
$darkShadow: 0 0 8px 5px rgba(0, 0, 0, 0.3);

File: tools/dashboard/src/App.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+body {
+	background-image: linear-gradient(180deg, rgba(38, 34, 80, 0.65) 0%, rgba(27, 25, 66, 0.85) 57.44%, #17163b 100%),
+		radial-gradient(50% 50% at 50% 50%, #201d47 0%, rgba(23, 21, 58, 0.91) 100%);
```
Comment (Romkin7) [inline tools/dashboard/src/App.scss]: How to reuse gradients: 
Screenshot 2021-06-25 at 1 31 33

**Review by Romkin7 — CHANGES_REQUESTED** [review]
Good Work, just few things to change for better development expereince in future for us.

File: tools/assets/sass/_variables.scss (hunk 1 above)
Comment (Romkin7) [reply tools/assets/sass/_variables.scss]: you can also do like this: $midnightBlue80: rgba($midnightBlue, 0.8);

File: tools/dashboard/src/App.scss (hunk 2 above)
Comment (jualiasha) [reply tools/dashboard/src/App.scss]: changes done

**Review by Romkin7 — APPROVED** [review]

File: tools/assets/sass/shared-components/_dashboard-main.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+aside {
+	width: 30vw;
+	height: 100vh;
+	position: fixed;
+	z-index: 999;
+	top: 0;
+	left: 0;
+	background: $patrickblue-linear;
```
Comment (Romkin7) [inline tools/assets/sass/shared-components/_dashboard-main.scss]: don't use - in names, in js we will encounter problems, use 
1. $patrickblueLinear
or 2. $patrickblue_linear

**Review by Romkin7 — CHANGES_REQUESTED** [review]

**Review by Romkin7 — APPROVED** [review]
good work so far

**Pull request merged**